index.html

```diff
@@ -0,0 +1,275 @@
+        href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+</head>
+
+<body>
+    <header class="main-header">
+        <div class="header-container">
+            <a href="" class="logo text-color">Solomon</a>
+            <nav>
+                <span class="material-symbols-outlined text-color">
+                    menu
+                </span>
+            </nav>
+        </div>
+    </header>
+    <section class="section-1">
+        <div class="content-0">
+            <div class="heading-1-container">
+                <h1 class="heading-1 margin-24">Hi there, I'm Solomon</h1>
+            </div>
+            <h2 class="heading-2">I'm a software Developer</h2>
+            <p class="section-par">I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you
+                see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+            <ul class="social-media">
+                <li><a href="#"> <img src="socialmedia/github.png" alt="github"></a></li>
+                <li> <a href="#"> <img src="socialmedia/linkedin.png" alt="linkedIn"></a></li>
+                <li> <a href="#"><img src="socialmedia/Vectori.png" alt="angelist"></a></li>
+                <li> <a href="#"> <img src="socialmedia/twitter.png" alt="twitter"></a></li>
+                <li> <a href="#"> <img src="socialmedia/mail.png" alt="email"></a></li>
+            </ul>
+        </div>
+    </section>
+
+
+    <section class="recent-profesional">
+        <div class="gridd">
+             <div class="my-recent-works">
+                <div class="H1">
+                    <h1> MY RECENT WORK</h1>
+                </div>
+                <div class="line">
+                    <img src="image/Vector.png" alt="line">
+                </div>
+                <div class="main-image">
+                    <img src="image/image1.png" alt="main image">
+                </div>
+                <div class="H2">
+                    <h2> Multi-post Stories</h2>
+                </div>
+                <div class="p-recent">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        has been the industry's standard dummy text ever since the 1500s, when an unknown 
+                        printer took a standard dummy text.
+                    </p>
+                </div>
+                <!-- <div>
+                    <div class="buttons-0">
+                        <button class="btn" type="button">CSS</button>
+                         <button class="btn" type="button">HTML</button>
+                         <button class="btn" type="button">Bootstrap</button>
+                        <button class="btn" type="button">Ruby</button>
+                    </div>
+                </div> -->
+                <ul class="buttons-0">
```

Comment:
- [ ] [OPTIONAL] Great job so far. I will suggest you remove unused codes from your codebase. They make your codebase to be untidy. We can always use git history to track old codes (comment codes).